stock_quant_manual_assign/tests/test_stock_quant_manual_assign.py

```diff
@@ -0,0 +1,187 @@
+# Copyright 2015 Oihane Crucelaegui - AvanzOSC
+# License AGPL-3 - See http://www.gnu.org/licenses/agpl-3.0.html
+
+from odoo.exceptions import ValidationError
+from odoo.tests.common import TransactionCase
+
+
+class TestStockQuantManualAssign(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
+        cls.quant_model = cls.env["stock.quant"]
+        cls.picking_model = cls.env["stock.picking"]
+        cls.location_model = cls.env["stock.location"]
+        cls.move_model = cls.env["stock.move"]
+        cls.quant_assign_wizard = cls.env["assign.manual.quants"]
+        cls.ModelDataObj = cls.env["ir.model.data"]
+        combo_product = cls.env["product.product"].create(
+            {
+                "name": "Combo Product Item",
+                "type": "consu",
+                "standard_price": 5.0,
+                "list_price": 10.0,
+            }
+        )
+        cls.product = cls.env["product.product"].create(
+            {
+                "name": "Test Combo Product",
+                "type": "combo",
+                "combo_ids": [(6, 0, [combo_product.id])],
```

Comment:
@mav-adhoc It was a luck that this worked. `combo_ids` model is `product.combo`. So, change the model here above.

Reply:
I was trying to use type consu instead but i was getting this error: "Quants cannot be created for consumables or services."

Reply:
There is a new property on products that allows to manage stock quantities. `is_storable` field.

Reply:
Done!